kimchi/src/circuits/polynomials/foreign_mul/foreign_mul_0.rs

```diff
@@ -0,0 +1,91 @@
+/// ForeignMul0 - Foreign field element constraints
+///
+///    Foreign field element F is comprised of three 88-bit limbs L0L1L2
+///
+///    * This circuit gate is used to partially constrain L0 or L1
+///    * The rest of L0 and L1 are constrained by a single ForeignMul2
+///    * It operates on the Curr row
+///
+/// Column | Curr
```

Comment:
Is it useful to have the layout here without talking about the constraints? I would prefer if you could move the description of the constraints here too.

Reply:
I think so.  So you can read the comment and understand each individually.  Added description of constraints.